tests/advl4tests/advl4_hostname_test.go

```diff
@@ -1078,15 +1091,16 @@ func TestAdvL4EndpointDeleteCreate(t *testing.T) {
 	// scale deployment to
 	g := gomega.NewGomegaWithT(t)
 
-	gwClassName, gatewayName, ns := "avi-lb", "my-gateway", "default"
-	modelName := "admin/abc--default-my-gateway"
+	gwClassName, gatewayName, ns := "avi-lb-14", "my-gateway-15", "default"
+	modelName := "admin/abc--default-" + gatewayName
+	svcName := "svc-15"
```

Comment:
Did you purposely miss svc-14 ?

Reply:
not on purpose. must have missed it. the indices are still unique